src/core/lib/iomgr/buffer_list.h

```diff
@@ -30,13 +30,86 @@
 #include "src/core/lib/iomgr/internal_errqueue.h"
 
 namespace grpc_core {
+
+/* A make-shift alternative for absl::Optional. This can be removed in favor of
+ * that once is absl dependencies can be introduced. */
+template <typename T>
+class Optional {
+ public:
+  void set(const T& val) {
+    value_ = val;
+    set_ = true;
+  }
+
+  bool has_value() { return set_; }
+
+  void reset() { set_ = false; }
+
+  T value() { return value_; }
+  T value_;
+  bool set_ = false;
+};
+
+struct ConnectionMetrics {
+  /* Delivery rate in Bps. */
```

Comment:
nit: In hindsight, this comment caused some confusions for users in internally. can you please change to Bytes/s?

Reply:
done